### PR TITLE
Update timer

### PR DESCRIPTION
**Pull request by AllentDan**
add batch size controller to the time counter.

### CODE REVIEW

**Comment by RunningLeon** [comment]
@AllentDan May fix ci errors

File: tools/test.py [hunk 1]
```diff
@@ -134,7 +134,8 @@ def main():
                 warmup=args.warmup,
                 log_interval=args.log_interval,
                 with_sync=with_sync,
-                file=args.log2file):
+                file=args.log2file,
+                batch_size=model_cfg.data.samples_per_gpu):
```
Comment (grimoire) [inline tools/test.py:138]: Not all models support batched inference.

Comment (AllentDan) [reply tools/test.py:138]: yes, that model would use default batch-size which is 1.

**Comment by AllentDan** [comment]
@RunningLeon @grimoire

**Review by grimoire — APPROVED** [review]

**Review by RunningLeon — APPROVED** [review]
LGTM

**Comment by grimoire** [comment]
If this fix is necessary for dev-1.x, please start another PR for 1.x.

**Comment by AllentDan** [comment]
> If this fix is necessary for dev-1.x, please start another PR for 1.x.

get you

**Pull request merged**